index.html

```diff
@@ -0,0 +1,302 @@
+	<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+	<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap" rel="stylesheet">
+</head>
+
+<body>
+	<section class="page">
+		<section class="container">
+			<header>
+				<ul>
+					<li><a class="myLogo font-style" href="">My Logo</a> </li>
+					<nav class="font-style">
+						<li><a class="toggler-icon"><img class="toggler-img" src="images/humburger.png"
+									alt="humberger-icon"></a>
+						</li>
+					</nav>
+
+				</ul>
+
+
+			</header>
+			<div class="main">
+				<section class="home-page">
+					<section class="intro-section">
+						<h1 class="main_heading font-style">I'm Fentahun <br> Glad to see you!</h1>
+						<p class="description font-style">
+							I’m a software developer! I can help you build a product , feature or
+							website Look through some of my work and experience! If you like what
+							you see and have a project you need coded, don’t hestiate to contact
+							me.
+						</p>
+					</section>
+					<div class="connect-section">
+						<h4 class="connect font-style">LET'S CONNECT</h4>
+						<ul class="socials">
+							<li>
+								<a class="social_medias twitter" href="#">
+									<img src="images/Vector (1).svg" alt="twitter logo">
+								</a>
+							</li>
+
+							<li>
+								<a class="social_medias" href="#">
+									<img src="images/Linkedin icon.svg" alt="Linkedin icon">
+								</a>
+							</li>
+							<li>
+								<a class="social_medias" href="">
+									<img src="images/mediumLogo.png" alt="Medium icon" class="medium" />
+								</a>
+							</li>
+							<li>
+								<a class="social_medias" href="#">
+									<img src="images/githublogo.png" alt="Github icon" />
+								</a>
+							</li>
+							<li>
+								<a href="#"><img src="images/Vector (2).svg" alt="two fingers"></a>
+							</li>
+						</ul>
+					</div>
+				</section>
+			</div>
+
+		</section>
+		<!-- project -->
+		<section id="portfolio" class="cards grid-container">
+			<section class="first-card card sec-div">
+				<img class="snapshoot-style" src="images/Snapshoot Portfolio.svg" alt="Snapshoot portfolio image">
+				<div class="sec-div">
+					<h2 class="card-header font-style on-small">
+						Tonic
+					</h2>
+					<div class="sec-div card-description">
+						<ul class="font-style">
+							<li class="card-list-title on-small"> CANOPY</li>
+							<li class="Job-title_and_year on-small">
+								Back End Dev
+							</li>
+							<li class="Job-title_and_year">
+								2022
+							</li>
+						</ul>
+					</div>
+					<p class="notice font-style">
+						A daily selection of privately personalized reads; no accounts or sign-ups required.
+					</p>
+					<ul class="programming-languages font-style">
+						<li>html</li>
+						<li>css</li>
+						<li>javaScript</li>
+					</ul>
+					<button id="0" class="project-button button" type="button">See Project</button>
+				</div>
+			</section>

```

Comment:
- [ ] You did a great job working on the previous reviewer's suggestions about the button styles. However, the applied changes introduced a new bug in your UI:

|Template|Your design|
|---|---|
| ![image](https://github.com/Fentahunn/finish_mobile_version/assets/77069011/cfa58656-c433-4952-989b-838401e3e281) | ![image](https://github.com/Fentahunn/finish_mobile_version/assets/77069011/dde609a1-97aa-4b57-967f-68967537f3aa) |

If you look closely, the spacing of the first card is different from the others. This is because it is the only card inside your grid-container. Please fix this and add some vertical spacing between each card as shown in the design guidelines! 🤓 🔧